File: app/src/main/java/vn/loitp/app/activity/customviews/textview/circletextview/CircleTextViewActivity.java
package vn.loitp.app.activity.customviews.textview.circletextview;

import android.os.Bundle;
import android.text.SpannableStringBuilder;

import com.core.base.BaseFontActivity;
import com.views.textview.circle.LCircleTextView;

import loitp.basemaster.R;

public class CircleTextViewActivity extends BaseFontActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        LCircleTextView textView = (LCircleTextView) findViewById(R.id.circle_text_view);
        textView.setTextSize(R.dimen.txt_18);

        SpannableStringBuilder builder = new SpannableStringBuilder(getResources().getString(R.string.hello_world));
        textView.setText(builder);
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_circle_textview;
    }

}

File: app/src/main/java/vn/loitp/app/activity/customviews/textview/circletextview/CircleTextViewActivity.kt
package vn.loitp.app.activity.customviews.textview.circletextview

import android.os.Bundle
import android.text.SpannableStringBuilder
import com.annotation.IsFullScreen
import com.annotation.LogTag
import com.core.base.BaseFontActivity
import kotlinx.android.synthetic.main.activity_text_view_circle.*
import vn.loitp.app.R

@LogTag("CircleTextViewActivity")
@IsFullScreen(false)
class CircleTextViewActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_text_view_circle
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        tvCircle.setTextSize(R.dimen.txt_medium)
        val builder = SpannableStringBuilder(resources.getString(R.string.hello_world))
        tvCircle.setText(builder)
    }

}
